Description: BlankOn rebranding

 Simply change the Gnome logo to BlankOn logo.
 .
 gnome-control-center (1:3.38.4-1blankon1) verbeek; urgency=medium
 .
   * Move to blankon revision number
Author: Herpiko Dwi Aguno <herpiko@gmail.com>
---

diff --git a/panels/info-overview/cc-info-overview-panel.ui b/panels/info-overview/cc-info-overview-panel.ui
--- a/panels/info-overview/cc-info-overview-panel.ui
+++ b/panels/info-overview/cc-info-overview-panel.ui
@@ -28,7 +28,7 @@
                   <object class="GtkImage">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="resource">/org/gnome/control-center/info-overview/GnomeLogoVerticalMedium.svg</property>
+                    <property name="resource">/org/gnome/control-center/info-overview/BlankOnLogoVerticalMedium.svg</property>
                   </object>
                   <packing>
                     <property name="expand">False</property>
